independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json

```diff
@@ -0,0 +1,843 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CQC Data",
+    "description": "To retrieve regulator review reports and ratings for England",
+    "version": "10",
+    "contact": {
+      "name": "Martyn Lesbirel",
+      "email": "ipc-author-cqc@outlook.com"
```

Comment:
@martynlesbirel Can you please add `website` property as well.

Documentation - [Adding metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
Website property added to CQC connector files

Reply:
@martynlesbirel The `website` or `url` is not showing up, Can you make sure that you have added